sdk/ml/azure-ai-ml/azure/ai/ml/entities/_deployment/online_deployment.py

```diff
@@ -517,6 +517,8 @@ class ManagedOnlineDeployment(OnlineDeployment):
     :type code_path: Union[str, PathLike], optional
     :param scoring_script: Scoring script name. Equivalent to code_configuration.code.scoring_script.
     :type scoring_script: Union[str, PathLike], optional
+    :param egress_public_network_access:
```

Comment:
Should we put little bit more documentation on what are the allowed values? Because this is more like Boolean with only two possible values.

Reply:
Thanks, updated the documentation to include allowed values.